### ISSUE TITLE
[bug]: chrome 34 "Cannot redefine property: name"

### ISSUE THREAD

**Issue by christophehurpeau** [Financial-Times/polyfill-library#179]
Url: https://cdn.polyfill.io/v3/polyfill.min.js?features=default,es2015,es2016,fetch&unknown=polyfill&flags=gated

User Agent: `Mozilla/5.0 (Linux; Android 5.0; SAMSUNG SM-G900F-ORANGE Build/LRX21T) AppleWebKit/537.36 (KHTML, like Gecko) SamsungBrowser/2.1 Chrome/34.0.1847.76 Mobile Safari/537.36`

Error: 

```
TypeError: Cannot redefine property: name
  at Function.defineProperty()
  at ? (/v3/polyfill.min.js:76:201)
  at ? (/v3/polyfill.min.js:76:352)
  at ? (/v3/polyfill.min.js:145:7329)
```

**Comment by JakeChampion** [comment]
This has been fixed in #181 and released in 3.33.0 -- https://github.com/Financial-Times/polyfill-library/releases/tag/v3.33.0

**Comment by christophehurpeau** [comment]
Thank you :)
